libs/challenge-registry/challenge-search/src/lib/challenge-search-filters.ts

```diff
@@ -0,0 +1,59 @@
+import { Filter } from '@sagebionetworks/challenge-registry/ui';
+import {
+  challengeStartYearRangeFilterValues,
+  challengeStatusFilterValues,
+  challengeDifficultyFilterValues,
+  challengeInputDataTypeFilterValues,
+  challengeSubmissionTypesFilterValues,
+  challengeIncentiveTypesFilterValues,
+  challengePlatformFilterValues,
+} from './challenge-search-filters-values';
+
+export const challengeStartYearRangeFilter: Filter = {
+  queryName: 'startYearRange',
+  label: 'Challenge Year',
+  values: challengeStartYearRangeFilterValues,
+  collapsed: false,
+};
+
+export const challengeStatusFilter: Filter = {
+  queryName: 'status',
+  label: 'Status',
+  values: challengeStatusFilterValues,
+  collapsed: true,
```

Comment:
@vpchung  I have added a new file used to config the filter(or filter panel). The `collapsed` means whether the filter will be auto open/collapsed by default.

Reply:
Let me know if you would like to make status open by default.

Reply:
Whoops!  I commented in the other convo.

Yes!  Let's do `year`, `status,` and `input data type` -- based on the listening tours, I think our users will likely use these facets first.